### DISCUSSION TITLE
Is there any options or any way to change color of horizontal only one plot line??

### DISCUSSION THREAD

**Discussion by Dongseung** [nhn/tui.chart#740] [Q&A]
I want to change color of only one plot line such as data '0' plot line.
However, I could not find the option that I want.. 

한개의 plot라인 색상을 바꿀 수 있는 옵션이 있을까요??

ex) 
![image](https://user-images.githubusercontent.com/75294699/163505574-aaed3de4-4ed0-4a60-945a-3dc6ba2672af.png)

**Comment by jwlee1108** [comment]
@Dongseung 
You can refer to this [link](https://github.com/nhn/tui.chart/blob/main/docs/ko/common-plot.md#lines). Check it, please.
